plugins/module_utils/gitlab.py

```diff
@@ -34,6 +34,7 @@
 
 def auth_argument_spec(spec=None):
     arg_spec = (dict(
+        ca_path=dict(type='str'),
```

Comment:
Not sure about the `ca_path` option name, since this option has multiple names across modules :
- `ca_path`
- `ca_cert`
- `ca_certs`
- `feed_ca_cert`
- `rhsm_repo_ca_cert`

Reply:
`ca_path` is the preferred name, ansible-core's `uri` module and `url` lookup (and the `url` module utils) also use that name.